index.js

```diff
@@ -282,6 +282,10 @@ function _minWidth (col) {
   return minWidth
 }
 
+function getWindowWidth () {
+  if (process && process.stdout && process.stdout.columns) return process.stdout.columns
```

Comment:
super small nit, let's go with `typeof process === 'object'`, rather than just checking if `process` is truthy. Otherwise, this is looking great.